reference/shipping.v3.yml

```diff
@@ -397,6 +465,24 @@ components:
           readOnly: true
           format: date-time
           example: 2022-09-21T14:15:00+00:00
+    checkoutSettings:
+      title: checkoutSettings
+      description: Data about the checkout settings object.
+      type: object
+      x-internal: false
+      properties:
+        checkout:
+          $ref: '#/components/schemas/countryListStrategy'
+    countryListStrategy:
+      title: countryListStrategy
+      description: Data about the strategy of listing countries at checkout
```

Comment:
This might be redundant with the description for `country_list_strategy`

Reply:
Thanks Andrea, I believe this is now more descriptive.

Reply:
@payaalaee I'm actually thinking we could remove the sentence since the meaning is the same as the description for `country_list_strategy`. How does that sound?

Reply:
Thanks Andrea, it's removed!🙏